Sources/TuistGenerator/Utils/NamespaceGenerator.swift

```diff
@@ -0,0 +1,84 @@
+import PathKit
+import StencilSwiftKit
+import SwiftGenKit
+import TSCBasic
+import TuistSupport
+
+enum NamespaceType {
+    case assets
+    case strings
+
+    var templateFileName: String {
+        switch self {
+        case .assets:
+            return "xcassets.stencil"
+        case .strings:
+            return "strings.stencil"
+        }
+    }
+}
+
+private extension NamespaceType {
```

Comment:
Nitpick: What about moving this to the declaration of the enum and making it `private func parser` ? The compiler will be happier processing everything as part of the body parsing.